upf/flowtable_tcp.h

```diff
@@ -45,12 +45,9 @@ typedef enum tcp_event
 
 /* flow lifetime in seconds */
 static const int tcp_lifetime[TCP_F_STATE_MAX] = {
-  [TCP_F_STATE_SYN] = 15,
-  [TCP_F_STATE_SYNACK] = 60,
-  [TCP_F_STATE_ESTABLISHED] = 299,
-  [TCP_F_STATE_FIN] = 15,
-  [TCP_F_STATE_FINACK] = 3,
-  [TCP_F_STATE_RST] = 6
+  [TCP_F_STATE_SYN] = 15,          [TCP_F_STATE_SYNACK] = 60,
```

Comment:
If there is possibility to avoid line merge in such case it would be good (maybe with something similar to `// clang-format off`)

Reply:
So there is no automatic way at the moment https://github.com/llvm/llvm-project/issues/61560
we can go with clang-format off but personally this does not bother me